### ISSUE TITLE
bug(angular-output-target): incorrect path componentCorePackage name different than rootDir

### ISSUE THREAD

**Issue by razvangeangu** [ionic-team/stencil-ds-output-targets#65]
When components `rootDir` (e.g. core) and `componentCorePackage` (e.g. @ionic/core) have different names the current interface import generates:

`import { Checkbox as ICheckbox } from '../@ionic/core/dist/types/components/my-checkbox/my-checkbox';`

Adjusting the relative import to the `rootDir` will accommodate all cases and generate the right path.

```diff
-  const typePath = path.relative(componentCorePackage, cmpMeta.sourceFilePath).replace('../src', path.join(componentCorePackage, distTypesDir)).replace('.tsx', '');
+  const typePath = path.parse(path.join(componentCorePackage, cmpMeta.sourceFilePath.replace(path.join(rootDir, 'src'), distTypesDir)));

-    ? `import { ${cmpMeta.componentClassName} as I${cmpMeta.componentClassName} } from '${typePath}';`
+    ? `import { ${cmpMeta.componentClassName} as I${cmpMeta.componentClassName} } from '${path.join(typePath.dir, typePath.name)}';`
```

**Comment by reda-alaoui** [comment]
Could a release be published to integrate this fix?
Version 0.0.3 can't be used inside a stencil project using a scoped name (e.g. a project named @foo/bar)

**Comment by razvangeangu** [comment]
@reda-alaoui @jthoms1 [this PR](https://github.com/ionic-team/stencil-ds-output-targets/pull/82) must go for 0.0.4 to work properly - otherwise events will fire twice.

**As a temporary fix**, if you look at [this code in 0.0.3](https://github.com/ionic-team/stencil-ds-output-targets/blob/ae3414af8bad531edcab04c04930736fa62fbded/packages/angular-output-target/src/generate-proxies.ts#L79) it would be easy to integrate the following code in your `stencil.config.ts` file that will produce the right output.

```ts
/* TODO: remove once @stencil/angular-output-target@0.0.4 is released (https://github.com/ionic-team/stencil-ds-output-targets/pull/66) */
import path from 'path';
const pr = path.relative;
path.relative = (from, to) => {
  const relative = pr(from, to);
  if (from.includes('@foo/bar')) {
    return relative.replace(/^\.\.\//, '');
  }
  return relative;
};
```

**Comment by reda-alaoui** [comment]
@razvangeangu thank you, I will try the workaround :+1:
